src/olympia/devhub/views.py

```diff
@@ -882,6 +891,16 @@ def addons_section(request, addon_id, addon, section, editable=False):
             queryset=addon.addons_dependencies.all(), addon=addon,
             prefix='dependencies')
 
+    if section == 'technical':
+        try:
+            whiteboard = Whiteboard.objects.get(pk=addon.pk)
+        except Whiteboard.DoesNotExist:
+            whiteboard = Whiteboard(pk=addon.pk)
+
```

Comment:
Same here, this doesn't create the Whiteboard instance, I think you want to either do a whiteboard.save() afterwards or use https://docs.djangoproject.com/en/1.8/ref/models/querysets/#django.db.models.query.QuerySet.get_or_create?

Reply:
I think it's because he doesn't want to save the instance until the POST, and then when the whiteboard `is_valid` (line 936)

Reply:
hmm, potentially true. As I wrote, I wasn't sure but if that's the case then my comments are invalid and can be ignored

Reply:
Yes, what Andrew said. I don't want to save just yet, otherwise it would create empty database rows just for viewing the page.